raiden/tests/unit/test_upgrade.py

```diff
@@ -90,22 +102,30 @@ def test_sequential_version_numbers(tmp_path):
         storage.update_version()
         storage.conn.close()
 
-    with ExitStack() as stack:
-        stack.enter_context(patch(
-            'raiden.utils.upgrades.UPGRADES_LIST',
-            new=upgrade_functions,
-        ))
-        stack.enter_context(patch(
-            'raiden.constants.RAIDEN_DB_VERSION',
-            new=19,
-        ))
-        older_db_file = stack.enter_context(patch('raiden.utils.upgrades.older_db_file'))
-        older_db_file.return_value = old_db_filename
+    with monkeypatch.context() as m:
```

Comment:
Should this now also check up to 20?

Reply:
Should we change this every time we introduce a new version? i don't really think it matters as this is a generic test that is not tied to a specific version.